packages/demand-api/src/payment/PaymentService.ts

```diff
@@ -88,14 +98,23 @@ export class PaymentService implements Payment {
   }
 
   getAdyenPaymentMethods(params: PaymentMethodsParams) {
-    return this.http.post<PaymentMethodsResponse>(`${this.apiV3}/${this.url}/adyen/payments-methods`, params)
+    return this.http.post<PaymentMethodsResponse>(
```

Comment:
Instead of using `providerApiVersion` maybe it is better to add optional version parameter to method

Reply:
Same for other methods below